test/views/loadingAndUnloading/loading/LocationViewSpec.scala

```diff
@@ -16,25 +16,42 @@
 
 package views.loadingAndUnloading.loading
 
+import base.AppWithDefaultMockFixtures
+import config.{PhaseConfig, PostTransitionConfig, TransitionConfig}
 import forms.LocationFormProvider
 import models.NormalMode
 import org.scalacheck.{Arbitrary, Gen}
+import play.api.Application
 import play.api.data.Form
+import play.api.test.Helpers.running
 import play.twirl.api.HtmlFormat
 import viewModels.InputSize
 import views.behaviours.InputTextViewBehaviours
 import views.html.loadingAndUnloading.loading.LocationView
 
-class LocationViewSpec extends InputTextViewBehaviours[String] {
+class LocationViewSpec extends InputTextViewBehaviours[String] with AppWithDefaultMockFixtures {
 
   override val prefix: String = "loadingAndUnloading.loading.location"
 
   private val countryName = nonEmptyString.sample.value
 
-  override def form: Form[String] = new LocationFormProvider()(prefix)
+  private val transitionConfig     = new TransitionConfig()
+  private val postTransitionConfig = new PostTransitionConfig()
```

Comment:
Once you've injected the phase config into the form provider then you won't need these or some of the changes below